code/modules/mob/living/carbon/human/life/handle_mutations_and_radiation.dm

```diff
@@ -129,7 +129,7 @@
 				vomit()
 			if(prob(5*minor_rad_multiplier))
 				//Nosebleed
-				if(prob(15) && drip(1))
+				if(prob(15) && drip(1) && species.anatomy_flags & NO_BLOOD)
```

Comment:
did you test this at all? `drip` already returns 0 if the `species` has `NO_BLOOD`.

Reply:
I irradiated myself as a skeleton and didn't notice any nosebleeds, I didn't test if there were any before the change and I should have done that

Reply:
I reverted the nosebleed commit since it isn't necessary, I'll take a look to see if the problem is still occurring